spec/fixtures/command/args_command.rb

```diff
@@ -1,3 +0,0 @@
-class ArgsCommand < Rectify::Command
-  def initialize(_, _, _); end # rubocop:disable Lint/MissingSuper, Style/RedundantInitialize
-end
```

Comment:
This file was replaced by `spec/fixtures/commands/args_command.rb`, below. It's not shown up as a diff of the same file because the directory was also renamed for better use of Guard.